web/packages/teleterm/src/mainProcess/mainProcessClient.ts

```diff
@@ -134,21 +168,12 @@ export default function createMainProcessClient(): MainProcessClient {
         clusterProperties
       );
     },
-    subscribeToAgentUpdate: (rootClusterUri, listener) => {
-      const onChange = (
-        _,
-        eventRootClusterUri: RootClusterUri,
-        eventState: AgentProcessState
-      ) => {
-        if (eventRootClusterUri === rootClusterUri) {
-          listener(eventState);
-        }
-      };
-      const channel = 'main-process-connect-my-computer-agent-update';
-      ipcRenderer.addListener(channel, onChange);
-      return {
-        cleanup: () => ipcRenderer.removeListener(channel, onChange),
-      };
+    /**
+     * Signals to the windows manager that the UI has been fully initialized, that is the user has
+     * interacted with the relevant modals during startup and is free to use the app.
+     */
+    signalFrontendAppReadiness(args: { success: boolean }) {
```

Comment:
I don't think we use the term 'frontend' anywhere else in code, we rather refer to 'renderer'. I wonder if we should try to be more consistent here, OTOH `rendererApp` may sound a bit unusual :)

Reply:
Yeah, and it has to differentiate the app in question from how Electron uses the term "app". 🤔 React app maybe?

It's true that we don't say "frontend" anywhere but at least this means that this term uniquely describes the frontend app. ;) We could also go into the "UI"/"user interface" territory. UI is a weird acronym to include in function names (mostly because of how we approach casing in acronyms in JS). `signalUserInterfaceReadiness`?

Reply:
`signalUserInterfaceReadiness` sounds cool to me.